tekton/pipeline_run_test.go

```diff
@@ -214,4 +214,38 @@ var _ = Describe("PipelineRun", func() {
 				To(Equal(persistentVolumeClaim))
 		})
 	})
+
+	Context("When PipelineAsCode metadata is present in the Release", func() {
+		//
+		It("copies Annotations & Labels matching the PAC prefix", func() {
+			release.ObjectMeta.Annotations["foo"] = "bar"
+			release.ObjectMeta.Annotations[pipelinesAsCodeMetaPrefix+"/meow"] = "cat"
```

Comment:
maybe one more annotation or label test like `foo/ + pipelinesAsCodeMetaPrefix`?

Reply:
Oh good idea! 
The code checks for a beginning PAC prefix, so `"foo/" + pipelinesAsCodeMetaPrefix` should not ever copy. I've updated the test with that test case, and the code still works. Thanks! :+1: